packages/lit-dev-content/rollup.config.js

```diff
@@ -101,6 +97,27 @@ export default [
       }),
     ],
   },
+  {
+    name: 'SSR',
```

Comment:
When testing this manually, a `mods-[hash].js` was created next to the `rollupout/server/ssr.js` file. This contains the mods import that litdev-search requires. I guess it doesn't matter that there is a tiny bit of duplication for the overall benefit.

Reply:
I think this is fine since it's just at build time and the workaround would complicate the bundling imo